react/react1/week2/todo/src/TodoItem.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from "react";
+//{ description, deleteItem, id } is from the parent todolist component
+export function TodoItem({ description, deleteItem, id }) {
+  // change the className
+  const [className, setClassName] = useState("unchecked");
+  function changeStyle() {
+    setClassName((prev) => {
+      // prev = "unchecked"?"unchecked":"checked";
+      if (prev === "unchecked") {
+        return "checked";
+      } else {
+        return "unchecked";
+      }
+    });
+  }
```

Comment:
I think you rather want something like this:
```js
const [isDone, setIsDone] = useState(false);
const className = isDone ? 'checked' : 'unchecked';

function onDoneChanged() {
  setIsDone(!isDone);
}

...

<input type="checkbox" checked={isDone} onChange={onDoneChanged} />
```

Reply:
good way of doing it. But I am not familiar with this way. I will learn more about it and try to use this way in the future. Thanks for your professional advice. :)